Rules.md

```diff
@@ -169,22 +165,7 @@ Note: The unit has to be paid later that turn.
 
 ## Investing
 When investing (through movement, conquest or command token), tokens can go from the player's supply
-on the board. The maximum number of coins that may be invested in the region is indicated on the
-map.
-
-# Special abilities of powers
```

Comment:
I rather like these. The game is asymmetric by nature and these abilities can be tweaked to make a nation more adapted to its territory.
[In GoT, the combat cards are often adapted to the initial territory of each house.]